R/ci.R

```diff
@@ -0,0 +1,46 @@
+#' Confidence/Credible Interval
+#'
+#' Compute Confidence/Credible Intervals (CI) for Bayesian and frequentist models.
+#' Documentation is accessible for:
+#' \itemize{
+#'   \item Bayesian models
+#'   \item LM and GLMs
+#'   \item Mixed models
+#' }
+#'
+#'
+#' @param model A model.
+#' @param ... Arguments passed to or from other methods.
+#'
+#'
+#' @examples
+#' library(bayestestR)
+#'
+#' ci(rnorm(1000))
+#'
+#' \dontrun{
+#' library(rstanarm)
+#' model <- rstanarm::stan_glm(mpg ~ wt + cyl, data = mtcars)
+#' ci(model)
+#' ci(model, ci = c(.80, .90, .95))
+#'
+#' library(brms)
+#' model <- brms::brm(mpg ~ wt + cyl, data = mtcars)
+#' ci(model)
+#' ci(model, ci = c(.80, .90, .95))
+#' }
+#'
+#'
+#' @export
+ci <- function(model, ci = .90, ...) {
+  UseMethod("ci")
+}
+
+
+
+#' @rdname ci
+#' @export
+ci.numeric <- function(model, ci = .90, verbose = TRUE, ...) {
+  hdi(model, ci=ci, verbose=verbose, ...)
```

Comment:
I guess this is a placeholder? I think we can actually use something like `lapply(ci, stats::quantile, ...)` here, or not?

Reply:
Yes, it was just to see how can I expand on this method in parameters without redefining it

As for the stuff in itself, I was thinking about a `method` argument, with "hdi" or "quantile" as possible values?

Reply:
We already thought about the `method`-argument and a `ci()`-function. I like this implementation, so in the end, you can always end up calling `ci()`, even if you want `hdi()`, which gives more flexibility (still, I would keep the `hdi()` function because it's better to find then).

Reply:
I agree!